connectors/heygen/avatars.graphql

```diff
@@ -0,0 +1,79 @@
+# These are the necessary federation and connector directives you will be using
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.10")
+  @link(
+    url: "https://specs.apollo.dev/connect/v0.1"
+    import: ["@connect", "@source"]
+  )
+  # A @source directive defines a shared data source for multiple connectors.
+
+  @source(
+    name: "heygen-v2"
+    http: {
+      baseURL: "https://api.heygen.com/v2"
+      headers: [{ name: "X-API-KEY", value: "{$config.apiKey}" }]
+    }
+  )
+
+scalar Timestamp
+
+enum GroupType {
+  PUBLIC_PHOTO
+  PRIVATE
+}
+
+enum TrainStatus {
+  empty
+}
+
+type Avatar {
+  id: ID!
+  name: String
+  gender: String
+  previewImageUrl: String
+  previewVideoUrl: String
+}
+
+type AvatarGroup {
+  id: ID!
+  name: String!
+  createdAt: Timestamp
+  numLooks: Int
+  previewImageUrl: String
+  groupType: GroupType
+  trainStatus: TrainStatus
+}
+
+type Query {
+  listAvatars: [Avatar!]!
+    @connect(
+      source: "heygen-v2"
+      http: { GET: "/avatars" }
+      selection: """
+      $.data.avatars {
+        id: avatar_id
+        name: avatar_name
+        gender
+        previewImageUrl: preview_image_url
+        previewVideoUrl: preview_video_url
+      }
+      """
+    )
```

Comment:
It looks like `Avatar` is probably an entity? This isn't absolutely necessary but if someone wanted to extend the `Avatar` in another schema or subgraph, we would need to mark this as an entity. 

That could be done with adding `entity: true` into the `@connect`. This could probably be done with a couple other items in the `photo-avatars` as well. Let me know your thoughts, otherwise this looks good to go.